docs/src/guide/about.rst

```diff
@@ -20,9 +20,3 @@ This book was made using `Sphinx <http://sphinx.pocoo.org/>`_ and `vim
     In 2017 the libuv project incorporated the Nikhil's work into the official
     documentation and it's maintained there henceforth.
 
-Licensing
```

Comment:
Interesting, we don't seem to have any license stated, so I guess it's MIT at the moment, like the whole project. I think a CC license makes more sense, so I'll have a look at who contributed to docs/ and ask everyone if they are ok changing it.